packages/wdio-browserstack-service/src/service.ts

```diff
@@ -2,28 +2,61 @@ import got from 'got'
 import type { OptionsOfJSONResponseBody } from 'got'
 import type { Services, Capabilities, Options, Frameworks } from '@wdio/types'
 import PerformanceTester from './performance-tester.js'
-
+import aiSDK from '@browserstack/ai-sdk-node'
```

Comment:
Let's import all named exports from ai-sdk-node in a single file and export again, can prevent changes in case ai-sdk-node has changes to multiple file.